modules/_labs/combobox/react/lib/Combobox.tsx

```diff
@@ -209,10 +245,9 @@ const Combobox = ({
   }, []);
 
   useEffect(() => {
-    if (autocompleteItems) {
-      setShowingAutocomplete(autocompleteItems.length > 0 && isFocused);
-    }
-  }, [autocompleteItems, isFocused, value]);
+    const items = getInteractiveAutocompleteItems();
+    setInteractiveAutocompleteItems(items);
+  }, [autocompleteItems]);
```

Comment:
maybe getInteractiveAutocompleteItems should be moved inside a hook, since it is used only here

Reply:
Arn't hooks more for reuse? I am not sure the value of creating one for a one off, maybe I am missing something though.

Reply:
hmm, I guess my suggestion is coming from exhaustive-depth rule.
https://github.com/facebook/react/issues/14920

Reply:
Oh interesting maybe we should add this to the codebase? What do you think @NicholasBoll

Reply:
I think we should add hooks lint rules. I see we added the plugin, but didn't enable any rules

Reply:
@NicholasBoll added both hook linting rules and made the fixes.